tests/test_usecase.py

```diff
@@ -682,16 +691,40 @@ def test_conf(self):
         self.assertFileContentEqual(os.path.expanduser('~/.pubsrc'),
                                     str_fixtures.sample_conf)
 
-    def test_editor_abort(self):
+    def test_editor_aborts(self):
         with self.assertRaises(FakeSystemExit):
             cmds = ['pubs init',
-                    ('pubs add', ['abc', 'n']),
-                    ('pubs add', ['abc', 'y', 'abc', 'n']),
                     'pubs add data/pagerank.bib',
-                    ('pubs edit Page99', ['', 'a']),
+                    ('pubs edit Page99', ['', 'n']),
                    ]
             self.execute_cmds(cmds)
 
+    def test_editor_succeeds_on_second_edit(self):
+        cmds = ['pubs init',
+                'pubs add data/pagerank.bib',
+                ('pubs edit Page99', [
+                    '', 'y',
```

Comment:
You could stress it more than the empty string, I think. What about:
`test_editor_succeeds_on_third_edit` with the typical missing colon: `@misc{Page99, title="TTT" author="X. YY"}`?